modules/mtc-migrating-vms-known-issues.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// * migration_toolkit_for_containers/mtc-migrating-vms.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="mtc-migrating-vms-known-issues_{context}"]
+= Known issues
+
+The following known issues apply when migrating virtual machine (VM) storage: 
+
+* Migrating VM storage cannot switch to persistent volume (PV) mode.
```

Comment:
support switching volume mode since v1.8.6-6

Reply:
Right, and this documentation is for 1.8.6 so that limit is no longer there.